daemon/oci_linux.go

```diff
@@ -989,6 +989,17 @@ func WithSysctls(c *container.Container) coci.SpecOpts {
 	}
 }
 
+// WithConsoleSize sets the initial console size
+func WithConsoleSize(c *container.Container) coci.SpecOpts {
```

Comment:
At some point we should move these to a non-"linux" file, as most of these could be equally used for Windows (not needed for this PR, we can probably just rename the file at some point and update the Windows code)

Reply:
Moved this to a non-linux file. The Windows code doesn't set the spec with the oci.SpecOpts, so for now it can't really share this function with linux code.